README.md

```diff
@@ -709,6 +709,81 @@ Accept: application/json
 | *response.expectedCompletionTimestamp* | no        | The UNIX time stamp at which the Data Subject Request is expected to be completed |
 | *response.requestID*                   | no        | The request ID known to the destination system                                    |
 
+## Consent
+
+A Consent Request is initiated when a [Data Subject](#Subject) specifies or changes consent preferences.
```

Comment:
Is this the "magic consent" endpoint we discussed during our Fox kick off meeting?

Reply:
Negative.